finesse/hardware/data_file_writer.py

```diff
@@ -33,7 +34,7 @@ def _get_metadata(filename: str) -> dict[str, Any]:
     return {
         "encoding": "utf-8",
         "name": filename,
-        "datetime": datetime.now().astimezone().isoformat(),  # include timezone
+        "datetime": get_time().isoformat(),
```

Comment:
I'm not sure about this change. I know we're using UTC time everywhere else, but I deliberately wanted to save the localtime in the metadata (i.e. with timezone included) because I think that's useful information. Jon does sometimes take the equipment abroad; not sure if he changes the timezone when he does though.

While I can see the argument for using the same timesource here as for the actual data, I'd be inclined to leave it as is.